waypoint_x:
  - 2
  - 0
  - 0
waypoint_y:
  - 2
  - 2
  - 0